zulipterminal/ui_tools/views.py

```diff
@@ -1299,10 +1299,18 @@ def __init__(self, controller: Any, stream_id: int) -> None:
             footlinks_width,
             len(visual_notification.label) + 4,
         )
+        visual_notification = [visual_notification]
```

Comment:
This is a bit unclear since you're reassigning another value in the var and later assigning another value conditionally. Could we call it something else?

Reply:
Perhaps append `_setting`?